Common/Input/KeyCodes.h

```diff
@@ -261,7 +261,14 @@ typedef enum _keycode_t {
 	NKCODE_EXT_MOUSEBUTTON_4 = 1006,
 	NKCODE_EXT_MOUSEBUTTON_5 = 1007,
 	NKCODE_EXT_MOUSEWHEEL_UP = 1008,
-	NKCODE_EXT_MOUSEWHEEL_DOWN = 1009
+	NKCODE_EXT_MOUSEWHEEL_DOWN = 1009,
+
+	// Virtual reality controller motion
+	NKCODE_EXT_MOTION_UP = 1101,
+	NKCODE_EXT_MOTION_DOWN = 1102,
+	NKCODE_EXT_MOTION_LEFT = 1103,
+	NKCODE_EXT_MOTION_RIGHT = 1104,
+	NKCODE_EXT_MOTION_FORWARD = 1105,
```

Comment:
Should we separate these for right vs left controller?

Reply:
That would require some code rewritting because the left and right controller are two different devices.

I find it cool that the gesture could be done by more devices.

Reply:
Yeah you're right, it's better to keep one definition here, and to instead send different device code from left or right. Didn't think about that.